listenbrainz/webserver/static/js/src/utils.tsx

```diff
@@ -64,46 +62,62 @@ export async function searchForSpotifyTrack(
     return tracks[0];
   }
   return null;
-}
+};
 
-export function getArtistLink(listen) {
+const getArtistLink = (listen: any) => {
   const artistName = _.get(listen, "track_metadata.artist_name");
   const firstArtist = _.first(
     _.get(listen, "track_metadata.additional_info.artist_mbids")
   );
   if (firstArtist) {
     return (
-      <a href={`http://musicbrainz.org/artist/${firstArtist}`} target="_blank">
+      <a
+        href={`http://musicbrainz.org/artist/${firstArtist}`}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
         {artistName}
       </a>
     );
   }
   return artistName;
-}
+};
 
-export function getTrackLink(listen) {
+// TODO: remove this "any" when a listen type has been defined.
+const getTrackLink = (listen: any): JSX.Element | string => {
   const trackName = _.get(listen, "track_metadata.track_name");
   if (_.get(listen, "track_metadata.additional_info.recording_mbid")) {
     return (
       <a
         href={`http://musicbrainz.org/recording/${listen.track_metadata.additional_info.recording_mbid}`}
         target="_blank"
+        rel="noopener noreferrer"
       >
         {trackName}
       </a>
     );
   }
   return trackName;
-}
+};
 
-export function getPlayButton(listen, onClickFunction) {
+const getPlayButton = (listen: any, onClickFunction: () => void) => {
```

Comment:
Will convert this `listen: any` to `listen: Listen` when a proper type for listens is added.